@here/harp-mapview/lib/Tile.ts

```diff
@@ -604,6 +604,15 @@ export class Tile implements CachedResource {
             return;
         }
 
+        // creates an empty plane if there are no objects on it,
```

Comment:
Now I am writing for the third time:
Please fill the line length on 100 chars in the comments.